services/userService.js

```diff
@@ -1,6 +1,10 @@
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
```

Comment:
thid-party module과 custom module을 import해오는 부분 사이에 의도적으로 공백을 두어 구분해주면 좋을 것 같습니다.

```suggestion
const bcrypt = require('bcrypt')
const jwt = require('jsonwebtoken')

```

Reply:
반영했습니다!